### PR TITLE
Bug and training

### LINKED ISSUE #14: Create direct links to training
Create direct links from https://bede-documentation.readthedocs.io/en/latest/training/index.html to relevant events on https://n8cir.org.uk/events/.

### PR DESCRIPTION
**Pull request by MarionBWeinzierl**
Resolves
#14 - Direct links to training
#13 - Link to Power AI Slack
#7 - Refer to page with contact and Slack workspace info
#32 - User group section
#33 - Cryo-EM docs

### CODE REVIEW

**Comment by alisonrclarke** [comment]
Will need to fix the header CSS so that the new heading fits - I'll take a look.

File: training/index.rst [hunk 1]
```diff
@@ -27,6 +35,7 @@ General Training Material
 * `CUDA training at Oakridge (slides and lecture recording) <https://www.olcf.ornl.gov/cuda-training-series/>`_
 * Sheffield `One <http://gpucomputing.shef.ac.uk/education/sheffield_onedaycuda/>`_ or `Two <http://gpucomputing.shef.ac.uk/education/cuda/>`_ Day Introduction to CUDA Course (slides and labs)
 * `Sheffield Parallel Computing with GPUs taught module (slides, labs, lecture recordings) <https://paulrichmond.shef.ac.uk/teaching/COM4521/>`_ 
+* The `Power AI User Group Slack workspace <https://www.poweraiug.org/join/>`_ is very helpful for all sorts of questions about Power AI.
```
Comment (alisonrclarke) [inline training/index.rst:38]: Site seems to be down at the moment - presume that's a temporary glitch?

Comment (MarionBWeinzierl) [reply training/index.rst:38]: Hm, still is down. It would have hoped that it would be back up by today. :-/

Comment (MarionBWeinzierl) [reply training/index.rst:38]: I'll give it one or two more days, and otherwise remove this section.

Comment (alisonrclarke) [reply training/index.rst:38]: Could ping Ed Edmondson to ask about it?

Comment (MarionBWeinzierl) [reply training/index.rst:38]: Good point!

Comment (MarionBWeinzierl) [reply training/index.rst:38]: OK, I contacted him and he forwarded it, and it seems to be resolved now.

**Review by alisonrclarke — APPROVED** [review]
I'm happy with this now if you're happy with the CSS changes.

**Pull request merged**